src/java/picard/sam/markduplicates/MarkDuplicatesWithMateCigar.java

```diff
@@ -58,7 +58,32 @@
  */
 @CommandLineProgramProperties(
         usage = "Examines aligned records in the supplied SAM or BAM file to locate duplicate molecules. " +
-                "All records are then written to the output file with the duplicate records flagged.",
+                "All records are then written to the output file with the duplicate records flagged.<br /><br />" +
+                "" +
+                "An even better duplication marking algorithm that handles all cases including clipped and gapped" +
```

Comment:
Even better than what?

These documentation blocks should all be able to stand on their own independently of others. If there is a comparison being made, it should always be explicit, not implicit.